### ISSUE TITLE
Splash page (accessibility)

### ISSUE THREAD

**Issue by cmenedes** [timkeane/nyc-lib#38]
# Current Behavior
The initial dialogue in the splash window has no headings. The view map link does not have a button role.

# Expected Behavior
There should be at least 1 h1 in the splash window.
The map link inside the splash window & screen reader modal should have a button role.

# Failure Information (for bugs)

## Steps to Reproduce
1. Build nyc-lib as described in readme, navigate to dist/examples/ol/finderapp.html
2. Screen reader should recognize an h1 header in splash page and also announce that View Map link is in fact a button
